adddigits.py

```diff
@@ -0,0 +1,13 @@
+#https://leetcode.com/problems/add-digits/
+#Given an integer num, repeatedly add all its digits until the result has only one digit, and return it.
```

Comment:
nit: add a space between `#` and `Given` for comments

Reply:
also, python uses snake case, the file should called `add_digits.py` and the function should be called `add_digits`

https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables